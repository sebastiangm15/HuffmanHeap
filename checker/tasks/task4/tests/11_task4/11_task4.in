15
16 X14
17 X15
3 X1
6 X3
7 X4
5 X5
8 X6
10 X7
9 X8
13 X11
14 X12
4 X2
11 X9
12 X10
15 X13
7
X10
X3
X5
X7
X8
X4
X6
