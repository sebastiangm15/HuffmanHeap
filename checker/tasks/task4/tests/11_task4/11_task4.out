X1X2X4X13X14X15X6X8X7X5X3X9X10X11X12
